core-bundle/contao/templates/_new/component/_table.html.twig

```diff
@@ -124,7 +124,7 @@
                                     <td>
                                         {%- block table_body_cell_content %}
                                             {%- block table_cell_content %}
-                                                {{- cell|nl2br|insert_tag_raw -}}
```

Comment:
This adjustment should be made in the `table` content element, not the `_table` component.

Reply:
I disagree. HTML should be generally usable in tables, so the component seems the correct place to change this.

Reply:
Of course, from a spec perspective, the HTML table element can contain any flow content. That is true for nearly all elements. But HTML should generally be written in the templates itself and not come precomposed through the template parameters. You can still use the components for this (as a skeleton) and then overwrite the respective blocks. Assuming that the template parameters already contain HTML in general is a flawed concept to me.

Reply:
Still disagree. 🤷‍♂️

Reply:
Once we switch to output encoding having `|raw` in the component seems way too dangerous to me. Using `|raw` should be a special case and we should only do that in the content element template where we know that the input is already sanitized. In the component we don’t know where the input is coming from and should not trust it.

Reply:
I also agree that `|raw` should be in the more specific content element template, not the component - since the content element specifically allows HTML.

Reply:
Changed in da10289aff487ce7cd8436e4db79569953b4a1b0.